rdbi-core/pom.xml

```diff
@@ -13,42 +13,10 @@
     <artifactId>rdbi-core</artifactId>
     <name>rDBI-core</name>
 
-    <build>
-        <plugins>
-            <!-- shading dangerous libs cglib and asm -->
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-shade-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>shade-proxying-implementation</id>
-                        <phase>package</phase>
-                        <goals>
-                            <goal>shade</goal>
-                        </goals>
-                        <configuration>
-                            <artifactSet>
-                                <includes>
-                                    <include>cglib:cglib-nodep</include>
-                                </includes>
-                            </artifactSet>
-                            <relocations>
-                                <relocation>
-                                    <pattern>net.sf.cglib</pattern>
-                                    <shadedPattern>com.lithium.dbi.rdbi.shaded.net.sf.cglib</shadedPattern>
-                                </relocation>
-                            </relocations>
-                        </configuration>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
-
     <dependencies>
         <dependency>
-            <groupId>cglib</groupId>
-            <artifactId>cglib-nodep</artifactId>
+            <groupId>net.bytebuddy</groupId>
```

Comment:
neat, where did you find this library, it's like a drop in replacement of cglib.

Reply:
from the cglib homepage  - https://github.com/cglib/cglib

>IMPORTANT NOTE: cglib is unmaintained and does not work well (or possibly at all?) in newer JDKs, particularly JDK17+. If you need to support newer JDKs, we will accept well-tested well-thought-out patches... but you'll probably have better luck migrating to something like [ByteBuddy](https://bytebuddy.net/).

Reply:
actually i'm pretty sure i've run across it before - i think netty uses it a lot.